README.md

```diff
@@ -54,7 +74,18 @@ bots:
   overmind: screeps-bot-overmind
 ```
 
+## Troubleshooting
+- Help, my server is running but I can't connect.
+  - Follow the instructions for [screepsmod-auth](https://github.com/ScreepsMods/screepsmod-auth)
+- I can't push any code via `rollup` to my server.
+  - Make sure your `screeps.json` configuration in your project is set properly. 
+  - In your `email:` field, simply put in your `username`. Verify your password is the same as your `screepsmod-auth` setting.
+
+- My map is all red, I can't actually spawn in!
+  - Restart your server, check your configuration and follow the instructions for [screepsmod-admin-utils](https://github.com/ScreepsMods/screepsmod-admin-utils)
+  - Ensure your username is set to the `whitelist:` setting in `config.yml`
 
+## Resources & Docs
```

Comment:
```suggestion
```
This is actually not visible in the rendered markdown, it just lets us write `[screepsmod-auth]` and then it will automatically create the link :)